backend/app/controllers/api/v1/admin/users_controller.rb

```diff
@@ -28,7 +26,7 @@ def create
     private
 
     def user_params
-        params.permit(:id, :utorid, :email, roles: [])
+        params.permit(:id, :utorid, roles: [])
```

Comment:
Are we not storing email in user anymore? Why?

Reply:
Good question! We have the email of TAs through their applicant info, we have the email of instructors in the instructor table. I don't think it makes sense to duplicate it. The only one left out is `admin`s, but they don't get stuff emailed to them.